platformio.ini

```diff
@@ -62,7 +31,7 @@ arduino_core_2_7_4 = espressif8266@2.6.2
 arduino_core_3_0_0 = espressif8266@3.0.0
 arduino_core_3_2_0 = espressif8266@3.2.0
 arduino_core_4_1_0 = espressif8266@4.1.0
-arduino_core_3_1_2 = espressif8266@4.2.0
+arduino_core_3_1_2 = espressif8266@4.2.1
```

Comment:
Should we remove unused versions?

Reply:
IMO no, as you can still use those in `platformio_override.ini` if you choose so.

Reply:
IMO, if we only need them for `platformio_override.ini`, we should move them there as well.
But let's do it the way you like it better.

Reply:
> IMO no, as you can still use those in `platformio_override.ini` if you choose so.

Could they be added in `platformio_override.ini` instead if they are only needed there?

Reply:
Yes they could also be added in `platformio_override.ini`